.changeset/lazy-ducks-complain.md

```diff
@@ -0,0 +1,46 @@
+---
+'braid-design-system': minor
+---
+
+---
+updated:
+
+- ButtonIcon
+
+---
+
+Add optional `tooltipPlacement` prop to `ButtonIcon`
+
+The `tooltipPlacement` prop allows you to specify the placement of the tooltip to either `top` or `bottom`.
+The default value is `top`.
+
+**EXAMPLE USAGE:**
+
+```jsx
+<Stack space="gutter">
+  <Inline space="gutter" alignY="center">
+    <ButtonIcon
+      size="standard"
+      icon={<IconArrow direction={'up'}/>}
+      label="The tooltipPlacement is “top”"
+      id="size-1"
+      tooltipPlacement="top"
+    />
+    <Text tone="secondary" size="xsmall">
+      TOP
+    </Text>
+  </Inline>
+  <Inline space="gutter" alignY="center">
+    <ButtonIcon
+      size="standard"
+      icon={<IconArrow direction="down"/>}
+      label="The tooltipPlacement is “bottom”"
+      id="size-2"
+      tooltipPlacement="bottom"
+    />
+    <Text tone="secondary" size="xsmall">
+      BOTTOM
+    </Text>
+  </Inline>
+</Stack>
```

Comment:
Typically the Example Usage has been kept a little tighter to show what is different or changing. This is to try and keep the changelog/release notes more streamlined.

This could be as simple as:
```suggestion
<ButtonIcon
  tooltipPlacement="bottom"
/>
```

Thoughts?

Reply:
Ahh good idea!